int64 maxtime
int64 tree_index
---
float64 time
float64 dist2go
float64 average_distance
float64 box_distance
float64 damage
float64 energy